### ISSUE TITLE
Ability To Alter All Comic Paths To New Database

### ISSUE THREAD

**Issue by Budlyte** [mylar3/mylar3#826]
When moving databases from one OS to another, for example from Synology to Unraid, the comics paths all change.  Is there a quick way to massively alter all comics paths in Mylar, for example: "\\DISKSTATION1\public\Comics\215 INK\(2015) Enormous" to "/comics/215 INK/(2015) Enormous"?

**Environment (please complete the following information):**
 - OS: Synology to Unraid
 - Branch / commit: v0.5.5 (master)

**Comment by evilhero** [comment]
There are instructions in the wiki [here](https://github.com/mylar3/mylar3/wiki/Moving-to-a-new-filesystem).

There are also posts on the forums that you can find by searching for the 'locmove' term if you need more troubleshooting.

There is a GUI option in the works, but currently (as it was previoulsy) - the ``locmove`` option is the only option available.

**Comment by Budlyte** [comment]
Good golly, how did I miss that?  I'm pretty sure I've even read that post before.

Thanks E.H., you're the best.

**Comment by Budlyte** [comment]
I ended up not doing this, and should have added in the beginning that I moved to a Docker container. This was as easy to fix as mapping my old NAS share in the Windows server to the new Unraid path. Docker doesn't care, it just says "Yeah, sure. Here's the files."

Container Path: //DISKSTATION1/public/Comics/    <---- Old Path
Host Path: /mnt/user/Comics/     ,                            <---- New Path
